src/components/BaseButton.tsx

```diff
@@ -10,6 +10,12 @@ type ElementProps = {
   children?: React.ReactNode;
   href?: string;
   onClick?: React.MouseEventHandler<HTMLButtonElement> | React.MouseEventHandler<HTMLAnchorElement>;
+  onMouseDown?:
+    | React.MouseEventHandler<HTMLButtonElement>
+    | React.MouseEventHandler<HTMLAnchorElement>;
+  onPointerDown?:
+    | React.PointerEventHandler<HTMLButtonElement>
+    | React.PointerEventHandler<HTMLAnchorElement>;
```

Comment:
intention is on markets table, clicking the row should navigate to specific market, but clicking the favorite button should not (via e.stopPropogation)